src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsEnforcementSpec.groovy

```diff
@@ -473,4 +477,28 @@ class PriceFloorsEnforcementSpec extends PriceFloorsBaseSpec {
         assert response.seatbid?.first()?.bid?.collect { it.id } == [bidResponse.seatbid.first().bid.first().id]
         assert response.seatbid.first().bid.collect { it.price } == [floorValue]
     }
+
+    def "PBS shouldn't return error when price floor enforcement service enforce bid request"() {
```

Comment:
I can't understand what this test is checking by reading its name. Original PR says that floor enforcement shouldn't fail on bidder request absence.